### PR TITLE
Task 5 (s3 integration)

### PR DESCRIPTION
**Pull request by nullpr**
endpoint: 
GET - https://336hgjm02a.execute-api.eu-west-1.amazonaws.com/dev/import

POST - https://hboemveppg.execute-api.eu-west-1.amazonaws.com/dev/products

- [x]  +1 (for JS only) - async/await is used in lambda functions
- [x]  +1 (All languages) - importProductsFile lambda is covered by unit tests. (for JS only) [aws-sdk-mock](https://www.npmjs.com/package/aws-sdk-mock) can be used to mock S3 methods
- [x]  +1 (All languages) - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into a new folder in the same bucket called parsed, and then deleted from uploaded folder)

csv
title,description,count,price
csv1,csv desc,3,5
csv6,csv desc,3,12

### CODE REVIEW

**Comment by rpayeras** [comment]
All works fine! Could you provide the frontend url?

**Comment by nullpr** [comment]
> All works fine! Could you provide the frontend url?

FE:
link: https://d3o4a0lv7fi8p3.cloudfront.net/

**Comment by rpayeras** [comment]
Ouch something its wrong, the response after upload csv file using signed URL gives a 403 error, check policies of lambda and bucket to resolve it ;)

image

**Comment by nullpr** [comment]
> Ouch, something its wrong, the response after upload csv file using signed URL gives a 403 error, check policies of lambda and bucket to resolve it ;)
> 
> image
have I missed something from the requirements for this task? 
don't see any mentions of the FE part (; (https://github.com/EPAM-JS-Competency-center/cloud-development-course-initial/blob/main/5_integration_with_s3/task.md)

File: import-service/src/common/constatns.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const BUCKET_NAME = "rs-import";
```
Comment (ambroseus) [inline import-service/src/common/constatns.ts:1]: ^^ typo in file name `constatns` -> `constants`

**Review by ambroseus — APPROVED** [review]

**Pull request merged**